src/lib/util.js

```diff
@@ -0,0 +1,45 @@
+const chalk = require(`chalk`);
```

Comment:
Could this file live a level up in the src directory or is the lib directory a standard of some kind?